python/hail/expr/aggregators.py

```diff
@@ -1022,6 +1031,8 @@ def hist(expr, start, end, bins):
     agg = _to_agg(expr)
     if not is_numeric(agg._type):
         raise TypeError("'hist' expects argument 'expr' to be a numeric type, found '{}'".format(agg._type))
-    t = tstruct(['binEdges', 'binFrequencies', 'nLess', 'nGreater'],
-                [tarray(tfloat64), tarray(tint64), tint64, tint64])
+    t = tstruct(binEdges=tarray(tfloat64),
```

Comment:
names are not pythonic, i see the same throughout functions, so probably best to note that something needs to tackle this for 0.2 in another PR